web/Views/Admin/AdminDashboard.cshtml

```diff
@@ -0,0 +1,127 @@
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <div class="col-4">
+                    <h6>Admin Actions</h6>
+                    <a class="btn btn-primary" asp-controller="Admin" asp-action="GetAdminList">View Admins</a> <br/>
+                    <p></p>
+                    <a class="btn btn-primary" asp-controller="Landlord" asp-action="RegisterGet" asp-route-createUnassigned="true">Create Unassigned Landlord</a> <br/>
+                </div>
+            </div>
+        </div> 
+        }
+    else
+    {
+        <div>
+            <div class="row">
+                <div class="col-1 mt-4">
+                    <h1>
+                        <i class="bi bi-person-square"></i>
+                    </h1>
+                </div>
+                <div class="col-7">
+                    <div class="row">
+                        <div class="col-7">
+                            <h6>Hello</h6>
+                            <h2>@Model.CurrentUser.Name</h2>
+                            <h6>You are not an admin for the ChangeAhead Bricks&Hearts project.</h6>
+                            @if (Model.CurrentUser.HasRequestedAdmin)
+                            {
+                                <p>If you no longer believe that you should be an admin, please cancel your admin access request:</p>
+                                @using (Html.BeginForm("CancelAdminAccessRequest", "Admin", FormMethod.Post))
+                                {
+                                    <button type="submit" class="btn btn-danger">Cancel Admin Access Request</button>
+                                }
+                            }
+                            else
+                            {
+                                <p>If you believe that you should be an admin, please request admin access:</p>
+                                @using (Html.BeginForm("RequestAdminAccess", "Admin", FormMethod.Post))
+                                {
+                                    <button type="submit" class="btn btn-primary">Request Admin Access</button>
+                                }
+                            }
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    }
+}
+else
+{
+    <div>
+        <div class="row">
+            <div class="col-1 mt-4">
+                <h1>
+                    <i class="bi bi-person-square"></i>
+                </h1>
+            </div>
+            <div class="col-7">
+                <div class="row">
+                    <div class="col-7">
+                        <h6>Hello</h6>
+                        <h2>Welcome to the ChangeAhead Bricks&Hearts Project</h2>
+                        <h6>Please login with Google in order to access the site.</h6>
+                        <a class="btn btn-primary" href="@Url.Action("Google", "Login")">Login</a>

```

Comment:
I think which option is better depends on exactly how the Home page is set up - if nothing appears there until a user is logged in, using [Authorize] is fine (because a logged-out user must be trying hard in order to reach the page); but if there are links to 'Landlord Portal' , 'Admin Portal' etc. are freely visible, it's better to provide the user a nice page when they attempt to click through to these pages.
I'll leave it how it is for now, on the logic that it's easier to convert it to [Authorize] later than have to reconstruct the page view later.

Reply:
That makes sense 👍